libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -828,6 +820,9 @@ void AC_PosControl::update_xy_controller(float ekfNavVelGainScaler)
 
     // run horizontal position controller
     run_xy_controller(dt, ekfNavVelGainScaler);
+
+    // update xy update time
+    _last_update_xy_ms = now;
```

Comment:
Was this supposed to make a difference? Because this is still using the time taken at the beginning, so having it here or up there doesn't make any difference.